aiida/orm/utils/node.py

```diff
@@ -70,7 +71,14 @@ def load_node_class(type_string):
         entry_point_name = strip_prefix(base_path, 'nodes.')
         return load_entry_point('aiida.node', entry_point_name)
 
-    raise exceptions.EntryPointError('unknown type string {}'.format(type_string))
+    # At this point we really have an anomolous type string. At this point, storing nodes with unresolvable type strings
```

Comment:
There are two typos here:
- "anomalous" type string (nit-pick, I know)
- probably, it should be "At _one_ point, [...] was allowed"